nixos/modules/services/web-apps/hedgedoc.nix

```diff
@@ -1,998 +1,197 @@
 { config, lib, pkgs, ... }:
 
 let
-  inherit (lib) literalExpression mdDoc mkEnableOption mkIf mkOption mkPackageOptionMD mkRenamedOptionModule types versionAtLeast;
+  inherit (lib) mkOption types mdDoc literalExpression;
```

Comment:
`mdDoc` is a no-op starting with 23.11 (https://nixos.org/manual/nixpkgs/unstable/#function-library-lib.options.mdDoc)

Reply:
I thought these were required for unstable between 23.05 and 23.11, with the plan being to remove them all once 23.11 was released? Did it change recently?

Reply:
I don't actually know. Recently I was asked to remove `mdDoc` from my PR.

Reply:
I've been asked the opposite the past few months. I'll do some research to see if I can figure out which one it is.

Also, see Sandros comment about the inherit list. I believe the listed ones are used often enough to warrant them being there.

Reply:
I believe the original reason was the comment earlier up: if the file were to ever be backported, it would need `mdDoc`. This is probably not going to happen within the remaining time of 23.05, so I could just remove `mdDoc`. But at the same time, there's probably gonna be a big purge after 23.11 anyway, so I think I'll just let them stay for now.